src/afl-forkserver.c

```diff
@@ -538,6 +538,14 @@ void afl_fsrv_start(afl_forkserver_t *fsrv, char **argv,
            "handle_sigill=0",
            1);
 
+    /* LSAN, too, does not support abort_on_error=1. */
+
+    if (!getenv("LSAN_OPTIONS"))
+     setenv("LSAN_OPTIONS",
+            "exitcode=" STRINGIFY(LSAN_ERROR) ":"
+            "fast_unwind_on_malloc=0",
+            1);
+
```

Comment:
have you checked for other defaults that might hurt performance and do not help detect issues? eg. symbolize

Reply:
Documentation for LSAN_OPTIONS is.. limited:-). But it seems everything is disabled by default, according to https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer

However, while LSAN_OPTIONS can be used to pass other options which asan may use, e.g. `handle_sigill`, I do not think this would really matter in an LSAN way.

I can of course add the ones which have been added for, let's say, UBSAN_OPTIONS, if you want though.